zetaclient/evm/evm_client.go

```diff
@@ -1440,7 +1324,7 @@ func (ob *ChainClient) LoadDB(dbPath string, chain common.Chain) error {
 	return nil
 }
 
-func (ob *ChainClient) SetMinAndMaxNonce(trackers []types.OutTxTracker) error {
+func (ob *ChainClient) SetMinAndMaxNonce(trackers []crosschaintypes.OutTxTracker) error {
```

Comment:
Need comment for the function

Reply:
https://github.com/zeta-chain/node/pull/1836/commits/be3e8f9862d885050fb7ac19a3845e62bc8b27cd